crates/storage_impl/src/lib.rs

```diff
@@ -262,9 +262,8 @@ impl<T: DatabaseStore> KVRouterStore<T> {
             )
             .await
             .map(|_| metrics::KV_PUSHED_TO_DRAINER.add(&metrics::CONTEXT, 1, &[]))
-            .map_err(|err| {
+            .inspect_err(|_| {
```

Comment:
instead discarding the error, log there error also. Do the same for the below inspect_err also.